contrib/ansible/roles/common/tasks/system-proxy.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Proxy | Install system http proxy environment variable
+  lineinfile: dest=/etc/environment regexp=^HTTP_PROXY= line=HTTP_PROXY="{{ system_http_proxy }}"
```

Comment:
Is the issue that /etc/environment isn't picked up because these are launched by init/systemd, instead of launched directly by the user?  In which case, these wouldn't have any effect on anything other than kubectl?

In which case, I'm not sure about it, since we don't really do much/anything to get your kubectl to work...